app/src/main/java/com/sopt/clody/presentation/ui/writediary/screen/WriteDiaryViewModel.kt

```
@@ -169,6 +175,55 @@ class WriteDiaryViewModel @Inject constructor(
+                checkEmptyFieldsMessage()
+            }.onFailure {
+                ensureDefaultEntry()
+                _failureMessage.value = it.localizedMessage ?: UNKNOWN_ERROR
+                _showFailureDialog.value = true
+            }
+        }
+    }
+
+    fun saveDraftDiary(year: Int, month: Int, day: Int) {
+        viewModelScope.launch {
+            val date = String.format("%04d-%02d-%02d", year, month, day)
```

detekt (1.23.8)

[warning] String.format("%04d-%02d-%02d", year, month, day) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)